packages/cli/internal/pkg/cli/spec/project.go

```diff
@@ -9,3 +11,12 @@ type Project struct {
 	Data          []Data              `yaml:"data,omitempty"`
 	Contexts      map[string]Context  `yaml:"contexts,omitempty"`
 }
+
+func GetContext(projectSpec Project, contextName string) (Context, error) {
```

Comment:
Rather than have this be a standalone function, you simplify it by defining it as a method on the `Project` struct:

```
func (p Project) GetContext(contextName string) (Context, error) {
  contextSpec, ok := p.Contexts[contextName]
  //...
}
```

Reply:
Fixed